src/deleteEmployee.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  originalData,
+  fetchData,
+  fillentry,
+  firebaseData,
+  toast,
+} from "./script";
+import { api, skillList, overlay, dataDelModal } from "./constants";
+const confirmButton = document.querySelector(
+  ".confirm-button"
+)! as HTMLButtonElement;
+const delData = (index: number) => {
+  fetch(api + "/employee/" + index + ".json", {
+    method: "DELETE",
+  })
+    .then((res) => {
+      console.log(res, "successfully deleted!!!");
+      toast(false, "Succesfully deleted the employee");
```

Comment:
Extract all toast messages to constants. Add an object AppStrings and add it there

Reply:
extract toast message in other pages too